### DISCUSSION TITLE
[灌水]

### DISCUSSION THREAD

**Discussion by wenjiadashao** [share121/inter-knot#1174] [其他]
![Uploading u=1750082635,408445144&fm=253&app=138&size=b931,274&q=75&n=0&f=JPEG&fmt=auto&maxorilen2heic=2000000.jpeg…]()

**Comment by wenjiadashao** [comment]
为什么我的图片导入进来不显示啊

**Comment by SixRadium** [comment]
要等他 **_uploading_** 完，显示出图片才行
